source/includes/administration/_products.md

```diff
@@ -0,0 +1,207 @@
+### Product Catalogs
+
+The product catalogs determines the product configured for a service type and maybe specific connections.
+
+<!-------------------- LIST PRODUCT CATALOGS -------------------->
+#### List product catalogs
+
+`GET /product_catalogs
+
+Retrieves a list of product catalogs configured in the system.
+
+```shell
+# Retrieve product catalog list
+curl "https://cloudmc_endpoint/rest/product_catalogs" \
+   -H "MC-Api-Key: your_api_key"
+```s
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "mode": "ALL_CONNECTIONS_OF_TYPE",
+      "serviceType": "cloudca",
+      "connectionIds": [],
+      "name": {
+        "en": "fgsfdg",
+        "fr": "sfdg"
+      },
+      "changes": [],
+      "description": {
+        "en": "sdfg",
+        "fr": "sdg"
+      },
+      "id": "5ba5df7c-1e60-4d99-869d-dd3d97826b2e",
+      "categories": [
+        {
+          "name": {
+            "en": "sdfg",
+            "fr": "sdfg"
+          },
+          "id": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
+        }
+      ],
+      "products": [
+        {
+          "metricType": "GAUGE",
+          "unit": {
+            "unit": "HOUR",
+            "name": {}
+          },
+          "period": "HOUR",
+          "name": {
+            "en": "vCPU",
+            "fr": "vCPU"
+          },
+          "transformer": {
+            "type": "PROPORTIONAL_TO_TIME"
+          },
+          "id": "342f0d9b-3c3c-456d-b853-31d713b79e72",
+          "attribute": "RAWUSAGE",
+          "source": "1",
+          "filters": [],
+          "sku": "vCPU",
+          "categoryId": "fcd8908d-e1a0-41f2-9c38-fed1d1f77f18"
+        }
+      ]
+    }
+  ]
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the product catalog.
+`name`<br/>*Object* | The name object in each language for the product catalog (en, fr or es).
```

Comment:
```suggestion
`name`<br/>*Object* | The name object in each language for the product catalog (en, fr, or es).
```
technically correct without the comma, i just think its more explicitly clear

Reply:
f428ed5b76ae49b8bf4e644a9c5cd49c51399110